src/mesh/mesh_io.jl

```diff
@@ -202,5 +250,5 @@ function load_mesh!(mesh::ParallelTreeMesh, restart_file::AbstractString)
   # Partition mesh
   partition!(mesh)
 
-  return mesh
+  return nothing
```

Comment:
```suggestion
  return mesh
```
As far as I can tell, it is standard Julia practice to return the mutated object also as the function return value, to allow easier chaining of multiple functions. Is there a reason why you changed the return value to `nothing` here?

Reply:
Why is `nothing` returned in `calc_volume_integral!` and similar functions in `src/solvers/dg/dg_[123]d.jl` then?

Reply:
Because this is still from a time when I didn't know anything about typical Julia practices 😆 Thanks for changing it!